docs/dev/new_check_howto.md

```diff
@@ -88,187 +91,166 @@ A Check is a Python class with the following requirements:
+There are two basic types of tests: unit tests for specific functionality, and integration tests that execute the `check` method and verify proper metrics collection. Note that [pytest][6] and [tox][7] are used to run the tests.
 
 For more information, see the [Datadog Checks Dev documentation][8].
 
-The first part of the `check` method below retrieves two pieces of information we need from the configuration file. This is a good candidate for a unit test. Open the file at `my_check/tests/test_check.py` and replace the contents with something like this:
+The first part of the `check` method we wrote retrieves and verifies two pieces of information we need from the configuration file. This is a good candidate for a unit test. Open the file at `awesome/tests/test_awesome.py` and replace the contents with something like this:
 
 ```python
 import pytest
 
 # Don't forget to import your Integration!
-from datadog_checks.my_check import MyCheck
-from datadog_checks.errors import CheckException
+from datadog_checks.awesome import AwesomeCheck
+from datadog_checks.base import ConfigurationError
 
 
+@pytest.mark.unit
 def test_config():
-    c = MyCheck('my_check', {}, {}, None)
+    instance = {}
+    c = AwesomeCheck('awesome', {}, [instance])
 
     # empty instance
-    instance = {}
-    with pytest.raises(CheckException):
+    with pytest.raises(ConfigurationError):
         c.check(instance)
 
-    # only url
-    with pytest.raises(CheckException):
+    # only the url
+    with pytest.raises(ConfigurationError):
         c.check({'url': 'http://foobar'})
 
-    # only string
-    with pytest.raises(CheckException):
+    # only the search string
+    with pytest.raises(ConfigurationError):
         c.check({'search_string': 'foo'})
 
-    # this should be ok
+    # this should not fail
     c.check({'url': 'http://foobar', 'search_string': 'foo'})
 ```
 
-The scaffolding has already set up `tox` to run tests located at `my_check/tests`. Run the test:
+`pytest` has the concept of _markers_ that can be used to group tests into categories. Notice we've marked `test_config` as a `unit` test.
+
+The scaffolding has already been set up to run all tests located in `awesome/tests`. Run the tests:
 
 ```
-ddev -e test my_check
+ddev test awesome
 ```
 
-The test we just wrote doesn't check our collection _logic_ though, so let's add an integration test. We will use `docker-compose` to spin up an Nginx container and let the check retrieve the welcome page. Create a compose file at `my_check/tests/docker-compose.yml` with the following contents:
+The test we just wrote doesn't check our collection _logic_ though, so let's add an integration test. We will use `docker` to spin up an Nginx container and let the check retrieve the welcome page. Create a compose file at `awesome/tests/docker-compose.yml` with the following contents:
```

Comment:
```suggestion
This test doesn't check our collection _logic_ though, so let's add an integration test. Use `docker` to spin up an Nginx container and let the check retrieve the welcome page. Create a compose file at `awesome/tests/docker-compose.yml` with the following contents:
```

Reply:
`use` sounds too authoritative imo especially since we're not really telling them _what_ to do but rather _how_ we do it

Reply:
Wdyt?

Reply:
`We will use` is passive, so maybe `We use` to make it active?

Reply:
Sure! Update and I'll commit